emails/management/commands/get_delayed_emails_from_sqs.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+import sys
+
+from botocore.exceptions import ClientError
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+from emails.views import (
+    _sns_inbound_logic, validate_sns_header, verify_from_sns
+)
+
+
+logger = logging.getLogger('events')
+
+class Command(BaseCommand):
+    help = 'Fetches messages from SQS dead-letter queue and processes them.'
+
+    def handle(self, *args, **options):
+        self.exit_code = 0
+        try:
+            dl_queue = apps.get_app_config('emails').dl_queue
+        except ClientError as e:
+            logger.error('sqs_client_error: ', extra=e.response['Error'])
+            self.exit_code = 1
+            sys.exit(self.exit_code)
+
+        messages = dl_queue.receive_messages(
+            MaxNumberOfMessages=10, WaitTimeSeconds=1
+        )
+        while len(messages) > 0:
+            for message in messages:
+                json_body = json.loads(message.body)
+                verified_json_body = verify_from_sns(json_body)
+                topic_arn = verified_json_body['TopicArn']
+                message_type = verified_json_body['Type']
+                validate_sns_header(topic_arn, message_type)
+                _sns_inbound_logic(topic_arn, message_type, verified_json_body)
```

Comment:
We should catch the response from `_sns_inbound_logic` and check the 200 before we delete.

Reply:
This queue should have 0 messages in it after we recover from down-time and run this script. 

If we only delete on 200's, we'll leave messages in SQS where we would otherwise have responded 4xx to SNS and then dropped the email data completely. 

Scenarios where we respond with 4xx:
* SNS message type we don't support
* messages without `commonHeaders`
* address does not exist
* replies require premium account
* cannot find message contents from s3 (future)

No matter what happens in `_sns_inbound_logic`, we should delete the message from SQS so we're still operating with the same "we don't keep any of the email data" architecture we have with SNS.

Reply:
Note: I did update the code to NOT delete the message from SQS, but only in the case of an `AccountSendingPausedException` as [@moz-astults pointed out a specific likely rate-limiting scenario](https://github.com/mozilla/fx-private-relay/pull/1322#discussion_r742958158) that would cause an error different from the regular errors that would otherwise drop the email data.